packages/react-core/src/Authenticator/hooks/useAuthenticator/useAuthenticator.tsx

```diff
@@ -43,6 +44,12 @@ export default function useAuthenticator(
   // `fields` from updating with current form state on value changes
   const serviceSnapshot = service.getSnapshot();
 
+  // legacy `QRFields` values only used for SetupTOTP page to retrieve issuer information, will be removed in future
+  const QRFields = useMemo(
+    () => getQRFields(serviceSnapshot as AuthMachineState),
+    [serviceSnapshot]
+  );
```

Comment:
Unfortunately, I don't really see an easy way of keeping it in the existing fields variable. We'd have to explicitly not delete it, then have to make changes to Vue, React and Angular so the Form Fields components won't automatically try to create a new bogus form field component. On the other hand, if we keep it in `useAuthenticator` it will only affect React, and won't require anything additional.

I'm up for more suggestions if you know of an easier way. Otherwise, we could wait until the next breaking change to fix this.

Reply:
Why can't we just put the workaround in `SetupTOTP`?

Reply:
@calebpollman The problem is how we get it into the `SetupTOTP` component? It's not in the `fields` component passed from `useAuthenticator`. That's why it doesn't update the QR image for the customer.

Reply:
Per offline discussion, I have removed the `QRFields` from `useAuthenticator`. We are now grabbing the information in `getMachineFields` instead, and then deleting it in `FormFields` component, since it's not technically a field that needs to be displayed.

Reply:
Reverted - added back in `QRFields` after offline discussion with @calebpollman